### ISSUE TITLE
Installation Failed

### ISSUE THREAD

**Issue by bayonhealth** [timsutton/make-profile-pkg#6]
I've been using your tool to put together a solution for automating the installation of enrollment profiles for my Macs. Earlier today I did a proof of concept, created a package with your tools and the package successfully installed. I then created some more packages with your tools, uploaded them to Munki and now the packages are saying "Installation Failed" when I install them. This is happening on my own computer, where only just this morning it successfully installed, and also happening on a fresh VM I'm testing with.

I doubt that's enough information, but what would cause this installation to be failing?

**Comment by chasetb** [comment]
Can you run `/usr/local/munki/managedsoftwareupdate -vvv --installonly` on your computer? This will give you more detail about what is actually happening during the Munki run. Copy and paste the results to http://pastebin.com/ and provide a link. (Make sure to edit out any sensitive info like your repo URL, if the URL is public-facing)

BTW: I suppose you know that Munki2 can install profiles natively without the need to wrap them in package first, right?

**Comment by timsutton** [comment]
Sounds like you might be able to leave Munki out of this and just concentrate on the package install. A good place to look for any script errors is `/var/log/install.log`.

**Comment by bayonhealth** [comment]
Thanks Tim. Here’s the weird thing, I hoped on my laptop last night, created a new package and it installed just fine. Puzzled by this, I went back and tried to install the packages I created this morning and they installed just fine. I’ll keep an eye on it, but it looks good now. Also, thanks so much for creating this. I’ve been trying to find a way to automate deploying enrollment profiles and I had been spinning my tires until I found your tool!

Jon Taylor

Bayon
p: 612-424-4195
w: bayonit.com

Apple Consultants Network
Apple Certified Support Professional
Apple Sales Product Professional


On May 5, 2015 at 8:18:33 AM, Timothy Sutton (notifications@github.com) wrote:

Sounds like you might be able to leave Munki out of this and just concentrate on the package install. A good place to look for any script errors is /var/log/install.log.
